### PR TITLE
Support creating a table with table & column comment in BigQuery

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Support creating a table with table & column comment in BigQuery

## Documentation

(x) No documentation is needed.

## Release notes

(x) Release notes entries required with the following suggested text:

```markdown
# BigQuery
* Support creating a table with table and column comment. ({issue}`13105`)
```

### CODE REVIEW

**Review by raunaqmorarka — COMMENTED** [review]
lgtm, but someone more familiar with BQ should also take a look

File: plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java [hunk 1]
```diff
@@ -237,23 +239,23 @@ static String bytesToStringConverter(Object value)
         return format("FROM_BASE64('%s')", Base64.getEncoder().encodeToString(slice.getBytes()));
     }
 
-    public static Field toField(String name, Type type)
+    public static Field toField(String name, Type type, @Nullable String comment)
```
Comment (hashhar) [inline plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java:242]: Why `@Nullable` only here?

**Review by hashhar — COMMENTED** [review]
A question about annotations but impl looks good.

A semi-related question - how can we remove a table or column comment using Trino? By setting those to `NULL` or empty strings?

**Comment by ebyhr** [comment]
We use `NULL` to remove the comment in syntax level. As you already know, the implementation depends on the connector because remote database may not allow setting `NULL` in their comment (e.g. Kudu).

File: plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java (hunk 1 above)
Comment (ebyhr) [reply plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryType.java:242]: Added a missing annotation.

**Review by hashhar — APPROVED** [review]

**Pull request merged**